serverless.yml

```diff
@@ -0,0 +1,46 @@
+service: serverless-graphql-apollo
+
+frameworkVersion: ">=1.21.0 <2.0.0"
+
+provider:
+  name: aws
+  runtime: nodejs8.10
+  stage: dev
+  region: us-east-1
+  environment:
```

Comment:
Pulling these from environment variables from now. Another, more mature approach might be Amazon Systems Manager Parameter Store.
Env variables are nice and portable, and we while we are going with Travis it will be very easy to store encrypted env variables.